migrations/versions/0365_add_notification_failures.py

```diff
@@ -0,0 +1,27 @@
+"""
+
+Revision ID: 0365_add_notification_failures
+Revises: 0364_add_templatep2pchecklist.py
+Create Date: 2023-11-01 15:23:30.268549
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+revision = "0365_add_notification_failures"
+down_revision = "0364_add_templatep2pchecklist"
+
+
+def upgrade():
+    op.create_table(
+        "notification_failures",
+        sa.Column("notification_id", postgresql.UUID(as_uuid=True)),
+        sa.Column("body", postgresql.JSONB(astext_type=sa.Text()), nullable=True),
```

Comment:
> astext_type=sa.Text())

What was the purpose of this? Does this turn integers into text for comparison purposes? I would have expected it kept the appropriate data type.

Reply:
done